Readme.md

```diff
@@ -1,20 +1,32 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128579731/22.1.5%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E3188)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+
+# WinForms - How to hide unnecessary series from the chart (Pivot - Chart Integration) 
+
+This example demonstrates how handle the [PivotGridControl.CustomChartDataSourceRows](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.PivotGridControl.CustomChartDataSourceRows) event to hide unnecessary series from the Chart control connected to the Pivot Grid.
+
+![Pivot Grid - Chart integration](images/pivotgrid.png)
+
+## Example Overview
+
+The `CustomChartDataSourceRows` event parameter’s [e.Rows](https://docs.devexpress.com/WindowsForms/DevExpress.XtraPivotGrid.PivotCustomChartDataSourceRowsEventArgs.Rows) property allows you to access data that is passed to the Chart. The Pivot Grid supplies data to the attached Chart control in the form of a simple table. Each row of this table is a single Pivot Grid cell. From this row (the `PivotChartDataSourceRow` object), you can get cell values, the argument object generated from the corresponding field value, and the series name generated from the opposite field values. For each record, you can also obtain information about a Pivot Grid cell to which this record corresponds. The example demonstrates how to modify the Chart data source table to hide the speicifed Chart series.
```

Comment:
1) ...`and the series name generated from the opposite field values.`

For my understanding, what do you mean by "**opposite** field values"?

2) `For each record, you can also obtain information about a Pivot Grid cell to which this record corresponds.`

Please change "**a** Pivot Grid cell" to "**the** Pivot Grid cell".

3) `The example demonstrates how to modify the Chart data source table to hide the speicifed Chart series.`

"speicifed" is misspelled. Please change it to "specified".